server/test/unit/server/webservices/views/test_consumer_groups.py

```diff
@@ -0,0 +1,302 @@
+import os
+os.environ['DJANGO_SETTINGS_MODULE'] = 'pulp.server.webservices.settings'
```

Comment:
These statements should not be here. If you rebase against the latest-feature branch then your ./run-tests.py will set this for you